src/components/SecondaryPanes/SecondaryPanes.css

```diff
@@ -5,6 +5,17 @@
   white-space: nowrap;
 }
 
+/*
+  We apply overflow to the container with the commandbar.
+  This allows the commandbar to remain fixed when scrolling
+  until the content completely ends. Not just the height of
+  the wrapper.
+  Ref: https://github.com/devtools-html/debugger.html/issues/3426
+*/
+.secondary-panes--sticky-commandbar {
```

Comment:
I did this as an auxiliary class just to be certain we aren't affecting other nodes by accident. Just in case.